app/src/main/java/org/jellyfin/androidtv/ui/playback/AudioNowPlayingActivity.java

```diff
@@ -330,7 +335,13 @@ public void onProgress(long pos) {
         public void onQueueStatusChanged(boolean hasQueue) {
             if (hasQueue) {
                 loadItem();
-                updateButtons(mediaManager.getValue().isPlayingAudio());
+                //Make sure our initial button state reflects playback properly accounting for late loading of the audio stream
+                mLoopHandler.postDelayed(new Runnable() {
```

Comment:
I'm not a fan of delaying code, this currently happens a lot in the app and is really a sign of bad code. It introduces all kinds of race conditions and bad UX (buttons showing up later).

I'd prefer a solution that doesn't need a handler for this.